packages/xchain-bitcoin/src/client.ts

```diff
@@ -128,13 +132,16 @@ class Client implements BitcoinClient, XChainClient {
   }
 
   // Private function to get keyPair from the this.phrase
-  private getBtcKeys(_phrase: string): Bitcoin.ECPairInterface {
-    const network = this.getNetwork() == 'testnet' ? Bitcoin.networks.testnet : Bitcoin.networks.bitcoin
-    const seed = BIP39.mnemonicToSeedSync(_phrase)
-    const wif = WIF.encode(network.wif, seed, true)
-    // TODO (@junkai121) Use `xchainCrypto.getSeed` while fixing it https://github.com/xchainjs/xchainjs-lib/issues/88
-    // const seed = xchainCrypto.getSeed(_phrase)
-    return Bitcoin.ECPair.fromWIF(wif, network)
+  private getBtcKeys(phrase: string): Bitcoin.ECPairInterface {
+    const isTestnet = this.getNetwork() === 'testnet'
+
+    const network = isTestnet ? Bitcoin.networks.testnet : Bitcoin.networks.bitcoin
+    const derive_path = isTestnet ? this.derive_path.testnet : this.derive_path.mainnet
```

Comment:
How about to provide a `derivePath` method? If we make `derive_path` private, the users can still ask for the path.